doc/environment.yml

```diff
@@ -6,11 +6,15 @@ channels:
 
 dependencies:
   - python=3.10
+  - ipython
```

Comment:
please comment that the newly added dependencies are required for the experiment tracker example. we might move this to a new package and when that happens we want to easily spot the dependencies that we don't need anymore, just add something like

```yaml
- scikit-learn # required for experiment tracker feature
```

Reply:
Done!